lib/super_digit.rb

```diff
@@ -1,15 +1,16 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
You need to be specific on what n is.  This should be O(log10 n) or O(n) if you specify that n is the number of digits.